stl/inc/yvals_core.h

```diff
@@ -1335,6 +1335,10 @@
 // C++20
 #define __cpp_lib_atomic_value_initialization 201911L
 
+#ifdef __cpp_char8_t
+#define __cpp_lib_char8_t 201907L

```

Comment:
We should update the comment block to indicate that these features are implemented unconditionally(ish) now.